errs/errs.go

```diff
@@ -37,3 +37,14 @@ func NewExitError(message string) error {
 	}
 	return cli.NewExitError(message, -1)
 }
+
+func FilterErrors(errors ...error) cli.MultiError {
```

Comment:
Instead of `FilterErrors` perhaps this should just be `MultiError` as its a MultiError wrapper?